File: Basics/Conditional Statements/Exercise/07.py
budget = float(input())
count_video_card = int(input())
count_processor = int(input())
count_ram = int(input())

price_video_card = count_video_card * 250
price_processor = (price_video_card * 0.35) * count_processor
price_ram = (price_video_card * 0.1) * count_ram
all_cost = price_ram + price_processor + price_video_card

if count_video_card > count_processor:
    all_cost = all_cost * 0.85

money_left = abs(budget - all_cost)

if all_cost > budget:
    print(f"Not enough money! You need {money_left:.2f} leva more!")
else:
    print(f"You have {money_left:.2f} leva left!")

File: JS-Basics/Conditional Statements/Exercise/07.js
function solve(budget, videoCards, processors, ramSticks) {
    const videoCardPrice = 250;
    const videoCardsCost = videoCards * videoCardPrice;

    const processorPrice = videoCardsCost * 0.35;
    const processorsCost = processors * processorPrice;
    
    const ramPrice = videoCardsCost * 0.10;
    const ramCost = ramSticks * ramPrice;

    let totalCost = videoCardsCost + processorsCost + ramCost;

    if (videoCards > processors) {
        totalCost *= 0.85;
    }

    const difference = budget - totalCost;
    const result = difference >= 0
        ? `You have ${difference.toFixed(2)} leva left!`
        : `Not enough money! You need ${Math.abs(difference).toFixed(2)} leva more!`;

    console.log(result);
}
